api/src/main/java/me/lokka30/treasury/api/economy/transaction/EconomyTransactionImportance.java

```diff
@@ -0,0 +1,43 @@
+package me.lokka30.treasury.api.economy.transaction;
+
+/**
+ * Allocates a certain 'importance' to a {@link EconomyTransaction}, which could be useful
+ * information for the economy provider and consumers which process the transaction.
+ * For example, a stream of low-importance transactions can be discarded from the transaction
+ * history, which can save a significant amount of database space usage if there are plugins
+ * installed that frequently send low-importance transactions.
+ *
+ * @author Nemo_64, lokka30
+ * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
+ */
+public enum EconomyTransactionImportance {
```

Comment:
maybe this should be an interface to open it up. i'm fine if it stays like that tough, but i want to start a discussion about it.

Reply:
I can't think of a use for an interface here but I'm all ears to any potential upgrades.

Reply:
I kinda figured this was effectively what we had the optional label for. I.e. a job payment might be labelled "job completion" so all my "job completion" entries can be deleted after a week or something. I'm on the fence for the concept in general because I feel like the economy implementation should be worrying about its storage details, not us, but having a unified way to indicate importance is also good.

Reply:
guess we're leaving it as an enum. 

@Jikoo are you satisfied with the other changes?